TrackletAlgorithm/DummyMessageLogger.h

```diff
@@ -0,0 +1,40 @@
+#ifndef MessageLogger_MessageLogger_h
+#ifndef L1Trigger_TrackFindingTracklet_interface_Logger_h
```

Comment:
Why does this line "#ifndef L1Trigger_TrackFindingTracklet_interface_Logger_h" need the CMSSW directory path, given that DummyMessageLogger is not part of CMSSW?

Reply:
Good catch. I'll remove the path and update the name.